lib/common/gather_info.sh

```diff
@@ -69,10 +69,10 @@ function get_catalog_source() {
     local catalog_ns
 
     catalog_name=$(KUBECONFIG="$kube_conf" oc get catalogsource -A \
-                -o jsonpath='{.items[?(@.metadata.name=="submariner-catalog")].metadata.name}')
+                -o jsonpath='{.items[?(@.metadata.name=='+"$DOWNSTREAM_CATALOG_SOURCE"+')].metadata.name}')
```

Comment:
Please, modify the definition of the variable to the following:
`(@.metadata.name=="'"$DOWNSTREAM_CATALOG_SOURCE"'")].metadata.name}')`

It should be a double quotes, surrounded by a single quotes, surrounded again by a double quotes.